MathSearch.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+# wfLoadExtension( 'MathSearch' );
```

Comment:
I think this is the correct way to load the MathSearch extension (not require once)